content/applications/inventory_and_mrp/barcode/operations/barcode_nomenclature.rst

```diff
@@ -1,84 +1,240 @@
+table, which opens a :guilabel:`Create Rules` pop-up window to create a new rule.
+
+The :guilabel:`Rule Name` field is used internally to identify what the barcode represents.
+
+The :guilabel:`Sequence` field represents the priority of the rule; meaning the smaller the value,
+the higher the rule appears on the table.
+
+The barcode :guilabel:`Type` field represents different classifications of information that can be
+understood by the system (e.g., :guilabel:`Package`, :guilabel:`Lot`, :guilabel:`Location`,
+:guilabel:`Coupon`, etc.).
+
+The :guilabel:`Encoding` field specifies which encoding the barcode uses; this rule **only** applies
+if the barcode uses this specific encoding. The available :guilabel:`Encoding` options are:
+:guilabel:`EAN-13`, :guilabel:`EAN-8`, :guilabel:`UPC-A`, and :guilabel:`GS1-28`.
+
+The :guilabel:`Barcode Pattern` field represents how the sequence of letters or numbers is
+recognized by the system to contain information about the product. Sometimes, when a certain amount
+of digits are required, the number of `.` is shown. `N` represents whole number digits, and `D`
+represent decimal digits.
+
+.. example::
+   `1...` represents any 4-digit number that starts with 1.
+   `NNDD` represents a two digit number with two decimal points. For example, `14.25` is 1425.
+
+After filling in the information, click the :guilabel:`Save & New` button to save the rule, and
+instantly start creating another rule. Or, click :guilabel:`Save & Close` to save the rule, and
+return to the table of rules.
+
+.. _barcode/operations/default-nomenclature-list:
+
+Default nomenclature list
+=========================
+
+The table below contains Odoo's list of :guilabel:`Default Nomenclature` rules. Barcode patterns are
+written in regular expressions.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Rule Name
+     - Type
+     - Encoding
+     - Barcode Pattern
+   * - Price Barcodes 2 Decimals
+     - Priced Product
+     - EAN-13
+     - 23.....{NNNDD}
+   * - Discount Barcodes
+     - Discounted Product
+     - Any
+     - 22{NN}
+   * - Weight Barcodes 3 Decimals
+     - Weighted Product
+     - EAN-13
+     - 21.....{NNDDD}
+   * - Customer Barcodes
+     - Client
+     - Any
+     - 042
+   * - Coupon & Gift Card Barcodes
+     - Coupon
+     - Any
+     - 043|044
+   * - Cashier Barcodes
+     - Cashier
+     - Any
+     - 041
+   * - Location barcodes
+     - Location
+     - Any
+     - 414
+   * - Package barcodes
+     - Package
+     - Any
+     - PACK
+   * - Lot barcodes
+     - Lot
+     - Any
+     - 10
+   * - Magnetic Credit Card
+     - Credit Card
+     - Any
+     - %.*
+   * - Product Barcodes
+     - Unit Product
+     - Any
+     - .*
 
 .. note::
-        When the barcode pattern contains .*, it means that it can contain any
-        number of characters, those characters being any number.
+   When the :guilabel:`Barcode Pattern` contains `.*`, it means it can contain any number or type of
```

Comment:
Ah, looks like this note admonition addresses what I encountered in the screenshot above. Great addition 👍